src/main/frontend/src/code-quarkus/pickers/extensions-picker.tsx

```diff
@@ -262,6 +274,10 @@ export const ExtensionsPicker = (props: ExtensionsPickerProps) => {
     setKeyBoardActived(-1);
     setFilter(f);
   };
+  const getParams = (paramName: string): string | null => {
```

Comment:
I think this function can also be declared outside of `ExtensionsPicker`

Reply:
Like the answer from the conversation above, I changed the position from this method, in order to put the filterParam in a prop. (I changed to 'quarkus-project-utils.ts')